### ISSUE TITLE
Add Bitmovin analytics

### ISSUE THREAD

**Issue by tomdyson** [royaloperahouse/stream-tv-app#21]
Probably using https://github.com/bitmovin/bitmovin-analytics-collector-android?

At minimum we should record video ID and title, e.g. https://github.com/tomdyson/bitmovin-player-tests/blob/main/multiple-sources.html#L61-L64

**Comment by Phobos98** [comment]
@tomdyson @mark-weber @CSeal 
It is already configured and implemented and the Java part.

https://github.com/royaloperahouse/stream-tv-app/blob/main/src/screens/HomeScreen/PlayerScreen/index.tsx

The above link is an example of a configured player. Here you can find the configuration example (`analytics` parameter) on how to use this. You can check Bitmovin Analytics Dashboard and see the values from that file in there.

So, basically, the only thing that has to be done is to actually determine what exactly we want to put there (I mean what values should be inside the corresponding fields)

**Comment by tomdyson** [comment]
Thanks, @Phobos98, that's good to know.

We'll consider what options (if any) we need to add for the `customDataX` fields. For now the important thing is that we record the `videoId` and `title` correctly. Assigning this to @CSeal.

**Comment by CSeal** [comment]
the estimate is about 30m(already completed)

**Comment by CSeal** [comment]
done
